.mailmap

```diff
@@ -35,3 +35,58 @@ Martin Jäger <martin@libre.solar>  <17674105+martinjaeger@users.noreply.github.
 Armand Ciejak <armand@riedonetworks.com>  <armandciejak@users.noreply.github.com>
 Chunlin Han <chunlin.han@linaro.org> <chunlin.han@acer.com>
 chao an <anchao@xiaomi.com>
+Martí Bolívar <marti.bolivar@nordicsemi.no>
+Martí Bolívar <marti.bolivar@nordicsemi.no> <marti.bolivar@linaro.org>
+Martí Bolívar <marti.bolivar@nordicsemi.no> <marti.f.bolivar@gmail.com>
+Martí Bolívar <marti.bolivar@nordicsemi.no> <marti@foundries.io>
+Martí Bolívar <marti.bolivar@nordicsemi.no> <marti@opensourcefoundries.com>
```

Comment:
wow, this guy has lots of identities :-D

Reply:
He can't seem to hold on to a job